addons/pos_self_order/static/tests/tours/mobile/self_order_after_meal_cart_tour.js

```diff
@@ -86,8 +86,8 @@ registry.category("web_tour.tours").add("self_order_after_meal_cart_tour", {
             { type: "select", name: "Fabric", value: "Custom" },
         ]),
         PosSelf.action.clickPrimaryBtn("Review"),
-        PosSelf.check.isOrderline("Desk Organizer", "5.87", "kidding", "M, Leather"),
-        PosSelf.check.isOrderline("Desk Organizer", "11.73", "okkk", "L, Custom"),
+        PosSelf.check.isOrderline("Desk Organizer", "5.87", "kidding", "Leather, M"),
+        PosSelf.check.isOrderline("Desk Organizer", "11.73", "okkk", "Custom, L"),
```

Comment:
why did the order change? This should respect the order assigned in the backoffice

Reply:
It not change, just the order of attributes

Reply:
exactly, the attributes have a specific order given in the backoffice. Why change it?

Reply:
They will have the same order in the backend as in the frontend, I have to do this because otherwise the arguments are in a random order depending on the order in which the options are chosen.

Reply:
![image](https://github.com/odoo/odoo/assets/118446179/e4dabf9e-3b96-46c4-be9a-cd334649e8c2)
it's weird to have the attribute order depend on the selection order

Reply:
Its fixed